src/server/lib/templates/header.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const styles = {
+  banner: {
+    width: "100%"
+  }
+};
+
+const Header: React.FC = () => {
+  return <img style={styles.banner} src="https://i.imgur.com/xxxV8Jo.png" />;
+};
```

Comment:
can someone show me how to move this banner image to cloud storage?

Reply:
TODO: add to "spoke rewired assets" bucket once [this](https://github.com/politics-rewired/unified-assemble-infra/pull/223) is merged